app/classifier/classifier.spec.js

```diff
@@ -215,4 +203,78 @@ describe('Classifier', function () {
       });
     });
   });
+
+  describe('with feedback enabled', function () {
+    let feedbackInitSpy;
+    let feedbackUpdateSpy;
+
+    before(function () {
+      feedbackInitSpy = sinon.spy();
+      feedbackUpdateSpy = sinon.spy();
+    });
+
+    beforeEach(function () {
+      const feedback = {
+        active: true
+      };
+      const actions = {
+        feedback: {
+          init: feedbackInitSpy,
+          update: feedbackUpdateSpy
+        }
+      };
+      wrapper = shallow(
+        <Classifier
+          classification={classification}
+          subject={subject}
+          feedback={feedback}
+          actions={actions}
+        />
+      );
+      wrapper.instance().componentDidMount();
+    });
+
+    afterEach(function () {
+      feedbackUpdateSpy.resetHistory();
+    });
+
+    it('should update feedback when an annotation is created', function () {
```

Comment:
Change the wording here to make it explicit that it tests what happens when the task changes?